src/graphics/program-lib/chunks/lit/frag/refractionDynamic.js

```diff
@@ -0,0 +1,58 @@
+export default /* glsl */`
+uniform float material_refractionIndex;
+uniform float material_invAttenuationDistance;
+uniform vec3 material_attenuation;
+uniform mat4 matrix_viewProjection;
+uniform mat4 matrix_model;
+uniform sampler2D uSceneColorMap;
+uniform vec4 uScreenSize;
```

Comment:
I wonder about these 4 typically global uniforms being added to this chunk. How do we handle other chunks including those as well?